#
#
# patch "src/options_list.hh"
#  from [0462e302b89179f4acb28ecb91f4255140d4a4a7]
#    to [27c44e89ebc9a828ac0a7b91505f997884646062]
#
============================================================
--- src/options_list.hh	0462e302b89179f4acb28ecb91f4255140d4a4a7
+++ src/options_list.hh	27c44e89ebc9a828ac0a7b91505f997884646062
@@ -381,9 +381,9 @@ SIMPLE_OPTION(move_conflicting_paths,
 SIMPLE_OPTION(move_conflicting_paths,
               "move-conflicting-paths/no-move-conflicting-paths",
               bool,
-              (F("move conflicting, unversioned paths into '%s' "
+              strdup((F("move conflicting, unversioned paths into '%s' "
                  "before proceeding with any workspace change") %
-                 bookkeeping_resolutions_dir).str().c_str())
+                      bookkeeping_resolutions_dir).str().c_str()))
 
 OPTSET_REL(globals, ssh_sign)
 SIMPLE_INITIALIZED_OPTION(ssh_sign, "ssh-sign", enum_string, "yes,no,only,check",
